genienlp/paraphrase/run_lm_finetuning.py

```diff
@@ -589,7 +589,8 @@ def main(args):
                                           cache_dir=args.cache_dir if args.cache_dir else None)
     tokenizer = tokenizer_class.from_pretrained(args.tokenizer_name if args.tokenizer_name else args.model_name_or_path,
                                                 do_lower_case=args.do_lower_case,
-                                                cache_dir=args.cache_dir if args.cache_dir else None)
+                                                cache_dir=args.cache_dir if args.cache_dir else None,
+                                                use_fast=False)
```

Comment:
Why use_fast=False? Without special tokens and special rules for preprocessing, fast should be good for all models, right?

Reply:
Yeah this change is extreme (mainly traumatized by past experiences with fast)
But I was seeing some weird tokenization issues for chinese with Marian (which is BART based).
I will add a command line flag to turn off fast.